util/topgen.py

```diff
@@ -985,6 +985,13 @@ def main():
     amend_clocks(topcfg)
     generate_clkmgr(topcfg, cfg_path, out_path)
 
+    # Hack alert
```

Comment:
@eunchan 
@vogelpi 

without this change, `top_englishbreakfast` generation fails since the number of alerts end up being wrong.

Reply:
I can imagine that this is not actually specific to `top_englishbreakfast` but necessary whenever the top is generated from scratch. I suspect we don't notice that for `top_earlgrey` because there all auto-generated files are checked in, i.e., the number of alerts is already known before calling topgen.

Reply:
@vogelpi could you have a look at #4944 when you get a chance?
I think that might solve this problem better... the solution there is STILL a hack..but at least it shouldn't require us to gingerly arrange all these generate functions...